__tests__/app.test.js

```diff
@@ -141,6 +141,30 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200: sorts articles by votes in ascending order when specified", () => {
+    return request(app)
+      .get("/api/articles?sort_by=votes&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles).toBeSortedBy("votes", { ascending: true });
+      });
+  });
```

Comment:
Nice first test, but ideally we should have two separate tests, to test each individual query and then a combined one like this